docker-example/README.md

```diff
@@ -1,37 +1,64 @@
-# Reflex Container Image Build
+# Reflex Docker Container
 
 This example describes how to create and use a container image for Reflex with your own code.
 
 ## Update Requirements
 
-The `requirements.txt` includes the reflex package which is need to install Reflex framework. If you use additional packages in your project you have add this in the `requirements.txt` first. Copy the `Dockerfile` and the `requirements.txt` file in your project folder.
+The `requirements.txt` includes the reflex package which is needed to install
+Reflex framework. If you use additional packages in your project you have to add
+this in the `requirements.txt` first. Copy the `Dockerfile`, `.dockerignore` and
+the `requirements.txt` file in your project folder.
 
-## Customize Reflex Config
+## Build Reflex Container Image
+
+To build your container image run the following command:
+
+```bash
+docker build -t reflex-app:latest . --build-arg API_URL=http://app.example.com:8000
+```
 
-The `rxconfig.py` includes the configuration of your Reflex service. Edit the file like the following configuration. If you want to use a custom database you can set the endpoint in this file. Ensure that `api_url` points to the publicly accessible hostname where the container will be running.
+Ensure that `API_URL` is set to the publicly accessible hostname or IP where the app
```

Comment:
If they don't set this, should we default to use localhost:8000 ?

Reply:
if the user doesn't provide `API_URL`, then the reflex default will be used, so whatever is in `rxconfig.py` or `http://localhost:8000`.

Do you think I should explicitly mention that in this section?